### PR TITLE
Update readme font/text support status

### PR DESCRIPTION
**Pull request by clach04**
examples/text makes use of:
  * fonts_get_system_font()
  * fonts_load_custom_font()
  * graphics_context_set_text_color()
  * graphics_draw_text()

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -180,20 +180,20 @@ Rocky.js currently implements a subset of Pebble's C-Style API. This section of
...
         <td>[Drawing Text](https://developer.pebble.com/docs/c/Graphics/Drawing_Text/)</td>
-        <td>Planned</td>
+        <td>Implemented</td>
         <td></td>
     </tr>
-    <tr class="planned">
+    <tr class="partial">
         <td>[Fonts](https://developer.pebble.com/docs/c/Graphics/Fonts/)</td>
-        <td>Planned</td>
+        <td>Partial</td>
```
Comment (franc0is) [inline docs/readme.md]: I think this is complete actually. What do you see missing?

Comment (clach04) [inline docs/readme.md]: My hunch was it was complete, just with some additional apis for embedded fonts. I'd rather a Pebbler make that call though :)

Comment (franc0is) [inline docs/readme.md]: It's complete :). Update it and i'll merge!

**Comment by clach04** [comment]
@franc0is font status updated.

**Pull request merged**